example/home/test/test_general.py

```diff
@@ -315,64 +313,55 @@ def setUp(self):
         BlogPageFactory(parent=self.home, title="Test post 2", slug="post-two")
         BlogPageFactory(parent=self.home, title="Test post 3", slug="post-three")
 
-    def test_pages_search(self):
+    def _query_pages(self, limit=None, offset=None):
         query = """
-        query ($term: String, $limit: PositiveInt, $offset: PositiveInt) {
-            pages(searchQuery: $term, limit: $limit, offset: $offset) {
+        query ($limit: PositiveInt, $offset: PositiveInt) {
+            pages(contentType: "home.BlogPage", order: "title", limit: $limit, offset: $offset) {
```

Comment:
note to self: this is because we test page size rather than search

Reply:
I did this in a few places where I removed assertions that weren't salient to what was being tested like looking for specific ids etc.